proxies/shared/resources/jsc/helpers/validationChecks.js

```diff
@@ -120,3 +120,17 @@ const validateDob = (errors, fieldValue, fieldPointer) => {
     }
     return true
 }
+
+const validateOdsCode = (errors, fieldValue, fieldPointer) => {
+    if (
+        !isUndefined(fieldValue)
+        && (
+            fieldValue === null
+            || typeof fieldValue !== "string"
+        )
+    ) {
+        errors.push(invalidError(fieldPointer));
+        return false
+    }
+    return true
```

Comment:
this function is returning true/false but nothing is ever done with the return values - are they needed?

Reply:
Good point! It matches the pattern of the rest of the validators, but it does seem useless